### PR TITLE
Pbi #650

### PR DESCRIPTION
**Pull request by r-kilimov**
[Pbi url](https://sarinevso.visualstudio.com/DefaultCollection/SarineProfile/_workitems?id=650&_a=edit)

You can changing order of stone properties [here](http://api-5.qa.sarine.com/viewer/v1/139D33GQHY/PBI650C1). 

Some stone properties aren't shown for the specific stone. Bug #655 [https://sarinevso.visualstudio.com/DefaultCollection/SarineProfile/_workitems?id=665&_a=edit]

For use this in template.config add

"stoneProperties": [
 {"key": "carat", "order": 1, "weight": 3},
 {"key": "color", "order": 2},
 {"key": "clarity", "order": 3}
],

### CODE REVIEW

File: app/js/app.dashboard.js [hunk 1]
```diff
@@ -499,7 +502,12 @@
                     }
                 } else if (field === 'stoneProperties.carat') {
```
Comment (adica) [inline app/js/app.dashboard.js]: @r-kilimov 
no need for this if - you can run the "format" attr on all SP (if exist)

File: app/js/app.dashboard.js [hunk 2]
```diff
@@ -567,11 +575,37 @@
...
+                    dashboard: $('.dashboard'),
+                    propertiesContainer: $('[data-properties]'),
+                    tmpPropertiesContainer: $('<div/>')
                 };
 
             elements.tmpSlidesContainer.append(elements.sliderPagesContainer.find('> .slide'));
 
+            // Enable Stone Properties
+            if (spConfig) {
+                elements.tmpPropertiesContainer.append(elements.propertiesContainer.children('[data-property]'));
```
Comment (adica) [inline app/js/app.dashboard.js]: @r-kilimov 
why do we need this "tmpPropertiesContainer"? we can run this without it?

**Comment by adica** [comment]
@r-kilimov 
* please use template engine (inline-markup) for the markup to reduce duplication
* change the "weight" property to be "format" property (more generic and can be used in the future to other stone properties)

**Comment by vlad-snrg** [comment]
Fixed: 
 - added missing text for stone properties
 - added js handler for formatting stone properties using `format` property
 - added html markup for missing stone property

**Comment by adica** [comment]
Looks OK
tnx

**Pull request merged**